.ci/integrationTestDownstream.groovy

```diff
@@ -115,8 +117,10 @@ pipeline {
         withGithubNotify(context: 'All', isBlueOcean: true) {
           deleteDir()
           unstash "source"
-          dir("${BASE_DIR}"){
-            sh ".ci/scripts/all.sh"
+          fielbeat(output: "${dockerLogs}.log", archiveOnlyOnFail: true){
```

Comment:
```suggestion
          filebeat(output: "${dockerLogs}.log", archiveOnlyOnFail: true){
```

Typo?

Reply:
Undefined variable in the output argument?